dungeon/src/dslToGame/nativefunction/NativeInstantiate.java

```diff
@@ -1,4 +1,7 @@
-package runtime.nativefunctions;
+package dslToGame.nativefunction;
```

Comment:
das package`dslToGame` hatte ich in #1017 entfernt, da das ganze Projekt eigentlich "DSL to Game" ist, kannst du das noch umsortieren?

Reply:
done.